app/api/dao/mentorship_relation.py

```diff
@@ -1,11 +1,11 @@
-from datetime import datetime, timedelta
+is not Nonefrom datetime import datetime, timedelta
 
+from app.utils.decorator_utils import email_verification_required
```

Comment:
Wrong order of imports. Should be added before `from app.utils.enum_utils import MentorshipRelationState`.